### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted_YourFirstExtension_Keyboard: The tooltip for all the video controls is not accessible by keyboard which is available to the mouse users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2828]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.1.1, MAS 2.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "API" link.
3.	From left navigation, select "Get Started".
4.	Navigate to "Your First Extension" link.
5.	Navigate to the video controls and observe if the tooltip are accessible using keyboard or not.

**Actual:**
When users navigate to the video controls using keyboard the tooltip is not accessible to the keyboard users but when user hovers over these controls using mouse the tooltip appears and it is available to the mouse users.

**Expected:**
The tooltip content which is available for mouse users should also be able to the keyboard users when the video controls receive keyboard focus.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The keyboard users will not get the same information which is available to the mouse users.

**MAS Reference:**
MAS 1.1.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

MAS 2.1.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment for Reference:**
[Code.visualstudio.com_APIGetStarted_YourFirstExtension_Keyboard The tooltip is not accessible by keyboard.pptx](https://github.com/microsoft/vscode-docs/files/3338200/Code.visualstudio.com_APIGetStarted_YourFirstExtension_Keyboard.The.tooltip.is.not.accessible.by.keyboard.pptx)

**Comment by octref** [comment]
I see no "hover tooltip". When I tab to the video tag, all the controls are accessible through tabs.

![image](https://user-images.githubusercontent.com/4033249/60549027-7f0df180-9cd8-11e9-827c-03c871623729.png)
